project/Hooje/README.md

```diff
@@ -0,0 +1,74 @@
+Simple Parallel Decision Tree in C++
+====
+
+# Basic Information
+Traditional Machine Learning models are still effective ways for classification.
+And Decision Tree is one of them.
+
+Decision Tree is a series a node that start with one root node and extends to many nodes that can represents the classfication rules for the data.
+For a dataset, Decision Tree will calculate the impurity (GINI index, entropy...) of every possible split, and choose the best one to be the rule of root node and create two leaf nodes.
+For each node, use the same method to decide the rule and create its child nodes.
+
+
+# Problem to Solve
+Decision Tree is an effective model in many classification senerio. However, it takes so much time to run because it cannot be speedup by CUDA like Neural Network, especially when there are a lot of features in that Dataset.
+
+
+Thus, I want to implement a simple version of Decision Tree in C++ (can be used from Python) from scratch to reduce the execution time. 
+Besides, I will compare the results between with and without SIMD.
+After that, I will try to finish multi-thread version and compare it with them.
+
+# Prospective Users
+Those people who use traditional ML models can benefit from my Decision Tree. 
+They can spent less time waiting for the model to finish.
+
+# System Architecture
+There will be a Dataset class, which will include features and its value.
+And a Node class, which represents each node in the Tree and connects with its parent node and child node. 
+And a Decision Tree Class, which will be the whole structure of Decision Tree and provide functions of Deicsion Tree.
```

Comment:
How would you design the data structure for the decision tree?  Will it be a tree data structure, or something more friendly to cache?  You can go in any way.  But it will make the project scope clear to state how you plan to do it.